presto-main/src/main/java/com/facebook/presto/execution/TaskExecutor.java

```diff
@@ -59,6 +59,7 @@
 import java.util.concurrent.atomic.AtomicLongArray;
 import java.util.function.DoubleSupplier;
 
+import static com.facebook.presto.operator.Operator.NOT_BLOCKED;
```

Comment:
The build was broken when I put this PR up. The commits have now been merged, so they'll be gone after rebase.